deploy/helm/sumologic/Chart.yaml

```diff
@@ -26,7 +26,7 @@ dependencies:
     condition: falco.enabled
   - name: metrics-server
     version: 5.11.9
-    repository: https://charts.bitnami.com/bitnami
+    repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
```

Comment:
Could you mention in the description of the pull request why it is needed? 
It will be worthy to mention this in the description https://github.com/bitnami/charts/issues/10539#issue-1257888825, in the future it will be easier to understand why the helm repo was changed

Reply:
updated the description, thanks!